### ISSUE TITLE
Is it possible to define timeInForce for orders?

### ISSUE THREAD

**Issue by telenskyt** [ccxt/ccxt#24681]
### Description

E.g. in [binance API](https://developers.binance.com/docs/binance-spot-api-docs/rest-api/public-api-endpoints#new-order-trade), in creating a new order, there is an option `timeInForce`, with these values:

> GTC	- Good Til Canceled  - An order will be on the book unless the order is canceled.
> IOC	- Immediate Or Cancel  - An order will try to fill the order as much as it can before the order expires.
> FOK	- Fill or Kill  - An order will expire if the full order cannot be filled upon execution.

I cannot find this anywhere in the [documentation of `binance.createOrder`](https://docs.ccxt.com/#/exchanges/binance?id=createorder). Is it possible to specify `timeInForce` via the ccxt library?

### Code

```
  

```

**Comment by sc0Vu** [comment]
@telenskyt Yes, you can set timeInForce via ccxt. BTW, I think the links are broken, I'll fix this asap.

**Comment by telenskyt** [comment]
Thanks for the answer! But I didn't find how to set `timeInForce` in the createOrder in the documentation (the [binance page](https://docs.ccxt.com/#/exchanges/binance))! Can you please give me a link?

**Comment by carlotestor** [comment]
Yes — `timeInForce` is a unified parameter you pass in `params`:

```js
await exchange.createOrder ('BTC/USDT', 'limit', 'buy', 0.001, 50000, {
    'timeInForce': 'IOC',   // 'GTC' (default), 'IOC', 'FOK', 'PO' (post-only)
});
```

It's documented in the Manual under createOrder rather than on the per-exchange page: https://docs.ccxt.com/#/README?id=order-structure

You can check which values an exchange accepts programmatically:

```js
console.log (exchange.features['spot']['createOrder']['timeInForce']);
```

There are also shorthands for the common cases — `postOnly: true` and the `has['createPostOnlyOrder']` flag. Anything CCXT doesn't unify is still forwarded verbatim, so exchange-specific values work too.
